src/engine/renderer/tr_local.h

```diff
@@ -736,7 +736,7 @@ enum class dynamicLightRenderer_t { LEGACY, TILED };
 
 	struct VBO_t
 	{
-		char     name[ MAX_QPATH ];
+		char     name[ 96 ];
```

Comment:
Why magic number. is there a significance to it? Since it's shared, maybe define a constant?

Reply:
I made it bigger because 64 was too small for some names.

> Since it's shared, maybe define a constant?

In principle, I suppose they're not necessarily the same; I could make a VBO with very long names that doesn't correspond to any IBO. Anyway the field is only used for `/listvbos` so it is not that important. I added a comment explaining how the name is only for debugging.